modules/raft/src/main/java/org/apache/ignite/raft/jraft/core/Replicator.java

```diff
@@ -1640,11 +1645,14 @@ private void sendTimeoutNow(final boolean unlockId, final boolean stopAfterFinis
     }
 
     private void sendTimeoutNow(final boolean unlockId, final boolean stopAfterFinish, final int timeoutMs) {
-        final TimeoutNowRequest.Builder rb = TimeoutNowRequest.newBuilder();
-        rb.setTerm(this.options.getTerm());
-        rb.setGroupId(this.options.getGroupId());
-        rb.setServerId(this.options.getServerId().toString());
-        rb.setPeerId(this.options.getPeerId().toString());
+        TimeoutNowRequest rb = raftOptions.getRaftMessagesFactory()
```

Comment:
I'd renamed this one as well but github doesn't allow me to do this for all usages.